### DISCUSSION TITLE
Updating the data of the modal while open

### DISCUSSION THREAD

**Discussion by sskieller** [Blazored/Modal#412] [Q&A]
Hello,

Is it possible to update the data of the modal while it is open?

Consider the following example:
I open a CreateTeam modal, fill it with info and then press save.
I then want to wait with closing the modal until I get a proper success from the backend. I am however not calling the backend directly, and instead I am using Redux and effects for doing this. So instead of awaiting an event, I want to be able to update a parameter in the modal, to let it know that it has succeeded OR if it has failed, show an error and then NOT close the modal.

-> Modal opened
Parameter: Loading = false, Loaded = false, Success = false;

-> Modal submitted
Parameter: Loading = true, Loaded = false, Success = false;

-> Success returned from backend
Parameter: Loading = false, Loaded = true, Success = true;
Close the modal

-> Failure returned from backend
Parameter: Loading = false, Loaded = true, Success = false;
Keep the modal open and show an error.

**Comment by chrissainty** [comment]
It's up to you when you close the modal. There is no reason you can't execute whatever logic you wish and then decide what to do about the modal once it completes.
